src/displayapp/fonts/README.md

```diff
@@ -6,94 +6,35 @@
 
 ## Generate the fonts:
 
-* Open the [LVGL font converter](https://lvgl.io/tools/fontconverter)
-* Enter the settings for the font that you wish to convert
-* Click on Convert, download the file and place it in `src/DisplayApp/Fonts`
+* Make sure you have installed lv_font_conv, and it's in the PATH
+  * See `doc/buildAndProgram.md`
+* run the `generate.py` script, while in the folder.
+  * it has a `-h` flag to show usage, nontheless: `python generate.py fonts.json`
```

Comment:
I think this entire section can be removed now that the dependency note has been added elsewhere and generate.py shouldn't be run manually.